src/main/java/apoc/index/FreeTextSearch.java

```diff
@@ -57,12 +57,32 @@ private IndexStats(String label, String property, long nodeCount) {
     @Procedure
     @PerformsWrites
     @Description("apoc.index.addAllNodes('name',{label1:['prop1',...],...}) YIELD type, name, config - create a free text search index")
-    public Stream<IndexStats> addAllNodes(@Name("index") String index, @Name("structure") Map<String, List<String>> structure) {
+    @Deprecated
```

Comment:
actually the new method should be deprecated, as in 3.1. we'll have default parameters, and don't need the extra name anymore. :)